pkg/fleet/env/env.go

```diff
@@ -43,6 +44,21 @@ var defaultEnv = Env{
 	},
 }
 
+// ApmLibLanguage is a language defined in DD_APM_INSTRUMENTATION_LIBRARIES en var
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏
```suggestion
// ApmLibLanguage is a language defined in DD_APM_INSTRUMENTATION_LIBRARIES env var
```

Reply:
Also why do we need this type?

Reply:
I added for two reasons:
* Make it clear what the `ApmLibraries` field contains, because IMO map[string]string is not great
* Add a method to return the version tag without modifying the raw variable

Reply:
> Add a method to return the version tag without modifying the raw variable
Isn't it only for `ApmLibVersion`?

Not a blocker tbh, feel free to ignore